python/ray/experimental/state/api.py

```diff
@@ -1217,20 +1220,15 @@ def get_log(
         stream=True,
     ) as r:
         if r.status_code != 200:
-            raise RayStateApiException(r.text)
-        for bytes in r.iter_content(chunk_size=None):
-            bytes = bytearray(bytes)
-            # First byte 1 means success.
-            if bytes.startswith(b"1"):
-                bytes.pop(0)
```

Comment:
For removing the first but, can we do it only if a parameter is set to maintain backwards compatibility with old version of ray?

Reply:
Yeah, we could do that.